changelog.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
```

Comment:
Where is this file being used?

Reply:
It was a misunderstanding, removing this file.